pkg/ansible/roles/mongodb/defaults/main.yml

```diff
@@ -0,0 +1,34 @@
+---
+mongodb_version: "3.4"
+mongodb_package: "mongodb-org"
+apt_key_fingerprint: "0C49F3730359A14518585931BC711F9BA15703C6"
+apt_keyserver: "keyserver.ubuntu.com"
+distribution_name: "{{ansible_distribution|lower }}"
+distribution_codename: "{{ ansible_distribution_release|lower }}"
+distribution_major: "{{ ansible_distribution_major_version }}"
+server_arch: "amd64,arm64"
+
+# MongoDB Repos
+mongodb_apt_repo: "deb [arch={{ server_arch }}] http://repo.mongodb.org/apt/{{ distribution_name }} {{ distribution_codename }}/{{ mongodb_package }}/{{ mongodb_version }} {{'main' if ansible_distribution == 'debian' else 'multiverse'}}"
+mongodb_yum_base_url: "https://repo.mongodb.org/yum/{{ ansible_os_family|lower }}/$releasever/{{ mongodb_package }}/{{ mongodb_version }}/{{ ansible_architecture }}"
+mongodb_dnf_base_url: "https://repo.mongodb.org/yum/{{ ansible_os_family|lower }}/7/{{ mongodb_package }}/{{ mongodb_version }}/{{ ansible_architecture }}"
+
+# MongoDB running config
+mongodb_storage_path: /data/db
+mongodb_log_path: /var/log/mongodb
+mongodb_config_path: /data/configdb
+se_linux: "TODO: (muawiakh)"
+directories:
+  - "{{ mongodb_storage_path }}"
+  - "{{ mongodb_log_path }}"
+  - "{{ mongodb_config_path }}"
+
+# Docker configuration
+mongodb_default_port: 27017
+mongodb_docker_image: "mongo:3.4.1"
+mongodb_docker_name: "mongodb"
+mongodb_docker_published_ports:
+  - 172.17.0.1:27017:27017
+mongodb_docker_volumes:
+  - "/tmp/mongodb_docker/db:{{ mongodb_storage_path }}"
+  - "/tmp/mongodb_docker/configdb:{{ mongodb_config_path }}"
```

Comment:
A user recently changed this to `$HOME/...` instead of `/tmp/...` as he/she didn't want to lose data on system restarts.
We might wanna consider that in case a user does a `vagrant halt` and a `vagrant up` repeatedly.